res/parking/parking_manager.c

```diff
@@ -523,6 +523,7 @@ static int manager_park(struct mansession *s, const struct message *m)
 	const char *announce_channel = astman_get_header(m, "AnnounceChannel");
 	const char *timeout = astman_get_header(m, "Timeout");
 	const char *parkinglot = astman_get_header(m, "Parkinglot");
+	const char *parkingpos = astman_get_header(m, "Parkingpos");
```

Comment:
The XML documentation at the top of the file needs to be updated to include this, along with describing the behavior if there is a conflict.